code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -280,13 +280,13 @@
 				safe_thing = victim.wear_mask
 
 		//only humans can have helmets and glasses
-		if(ishuman(victim))
+		if(!ismonkey(victim))
```

Comment:
Why?
lizards/moths are a human subtype

Reply:
yeah iunno why it didn't work until I added this. It doesn't matter though since there is an earlier check.